src/components/TabsPage.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Tab } from '../types/Tab';
+
+type Props = {
+  tab: Tab[],
```

Comment:
The prop 'tab' is an array and hence should be named in plural form 'tabs' for better readability and understanding of the code.